### PR TITLE
adding editors test done

### PR DESCRIPTION
**Pull request by BiPhan4**

### CODE REVIEW

**Comment by nuggetnchill** [comment]
taking a look at these now 😃

**Comment by BiPhan4** [comment]
@nuggetnchill Hey Nug - you are the only reviewer now. Please merge this PR after you approve. Please let me know if something needs to change. Thank you.

**Review by nuggetnchill — COMMENTED** [review]
lgtm

**Review by nuggetnchill — APPROVED** [review]

**Pull request merged**
